server/tournaments/index.ts

```diff
@@ -1699,6 +1727,33 @@ const chatCommands: ChatCommands = {
 					}
 				}
 			}
+		} else if (cmd === 'export') {
+			if (room.lastTournamentData && room.lastTournamentData.type === 'tree') {
+				let tourString = '';
+				let tourData = [];
+				const tourWinner = room.lastTournamentData.rootNode.team;
+				let roundCounter = Math.ceil(Math.log(room.lastTournamentData.playersLength) / Math.log(2));
+				tourData.push(room.lastTournamentData.rootNode);
+				while (tourData.some(node => node.children)) {
+					const tourDataCopy: TournamentDataNode[] = tourData;
+					tourData = [];
+					for (const copyChild of tourDataCopy) {
+						if (copyChild.children) {
+							for (const child of copyChild.children) {
+								tourData.push(child);
+							}
+							const winner = (copyChild.team === copyChild.children[0].team);
+							tourString = '<span' + (winner ? ' style="font-weight:bold;"' : '') + `>${copyChild.children[0].team}</span> vs <span` + (!winner ? ' style="font-weight:bold;"' : '') + `>${copyChild.children[1].team}</span><br>` + tourString;
+						}
+					}
+					tourString = `<br><strong>Round ${roundCounter}</strong><br>` + tourString;
+					roundCounter -= 1;
+				}
+				tourString = `<p> Tournament winner: <strong>${tourWinner}</strong></p>` + tourString;
+				return this.sendReplyBox(`<div style="max-height: 250px; overflow-y: auto"><p style="font-weight:bold;">${Chat.escapeHTML(this.room.title)} elimination tournament paste</p>` + tourString + '</div>');
+			} else {
+				return this.errorReply("You can only use the tournamentexport command after an elimination tour in this room has ended");
```

Comment:
tournamentexport -> tournament export?

Reply:
also, why? why not support exporting the current state of the tournament if someone ran it in the middle?

Reply:
Oh, the function used to be called tournamentexport, forgot to change.
Is it useful to show the current state of the tournament? I thought the paste would be for finished tours.

Reply:
I dont know if its 'useful' per se, but in my mind the 'tour' command should be implicitly scoped to the current tour (which is how all the other subcommands work), and only if there is no current tour should it do something different. If we named this new command `lastexport` or `latestexport` or something it would make sense to always point to the past, but to me, the `tour` command should operate on the current tour by default.

Reply:
that makes sense, and that's why i had it as a separate chatcommand or something before. 
If you only want me to act on the current tour, are you saying that I shouldn't be able to access the tour after the tour ends?
thanks.

Reply:
I'd have written this so that if there is a current tour, just use the current (unfinished bracket data) bracket data, otherwise use the last saved bracket data